.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  # Only run push on master
+  push:
+    branches:
+      - master
```

Comment:
This is why it didn't run on your PR. I think we want this to be `on: [push, pull_request]`. I wouldn't bother with the markdown exclusion. PRs modifying only docs are rare and we don't care about wasting GitHub's money.

Reply:
No it doesn't run because I'm not a collaborator 😅. I use this configuration in another project and it works fine. Can remove the md bits though, it's more useful if/when you check in docs like mkdocs sites and have a longer CI time

Reply:
the master-only push is to reduce PR noise, as otherwise it'd run both for any collaborators pushing to branches directly

Reply:
That seems like what we want. I push branches to test things on CI all the time (i almost never run tests locally). Moreover, it means that people working in the repo will have their changes tested prior to merge. Plus it's how every other project is set up.

I don't see how this would build your branch even if you pushed your branch to the repo directly.

Reply:
ah! In that case I'll change it. Still fairly certain it won't run regardless though until it's merged

Reply:
4212163